pybamm/models/full_battery_models/base_battery_model.py

```diff
@@ -546,7 +563,7 @@ def __init__(self, extra_options):
 
         # Check options are valid
         for option, value in options.items():
-            if option in ["working electrode"]:
+            if option in ["half-cell"]:  # is this exception still necessary?
                 pass
```

Comment:
Let's see what @tinosulzer says, as he implemented the original `working electrode` model. If you remove this line, do all tests still work?